addons/project/static/src/js/project_list.js

```diff
@@ -111,7 +111,7 @@ const ProjectListController = ListController.extend({
         });
 
         if (allowContinue) {
-            Dialog.buttons.splice(0, 1,
+            dialog.buttons.splice(1, 0,
```

Comment:
We should probably fix this in 15, shouldn't we? 🙂

Reply:
indeed, we should :p